libraries/AC_AttitudeControl/AC_AttitudeControl.cpp

```diff
@@ -957,7 +958,7 @@ void AC_AttitudeControl::reset_yaw_target_and_rate(bool reset_rate)
         _euler_rate_target.z = 0.0f;
 
         // Convert euler angle derivative of desired attitude into a body-frame angular velocity vector for feedforward
-        euler_rate_to_ang_vel(_euler_angle_target, _euler_rate_target, _ang_vel_target);
+        euler_rate_to_ang_vel(_attitude_target, _euler_rate_target, _ang_vel_target);
```

Comment:
I think this is the only change in behavior, `_attitude_target` is updated in this function but `_euler_angle_target` was not updated to keep in sync.